continuous_integration/install.sh

```diff
@@ -44,70 +44,80 @@ print_conda_requirements() {
+            else
+                REQUIREMENTS="$REQUIREMENTS $PACKAGE==$PACKAGE_VERSION"
+            fi
         fi
+
     done
     echo $REQUIREMENTS
 }
 
-create_new_conda_env() {
-    # Skip Travis related code on circle ci.
-    if [ -z $CIRCLECI ]; then
-        # Deactivate the travis-provided virtual environment and setup a
-        # conda-based environment instead
-        deactivate
-    fi
-
-    # Use the miniconda installer for faster download / install of conda
-    # itself
-    wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
-        -O ~/miniconda.sh
-    chmod +x ~/miniconda.sh && ~/miniconda.sh -b
-    export PATH=$HOME/miniconda3/bin:$PATH
-    echo $PATH
-    conda update --quiet --yes conda
 
-    # Configure the conda environment and put it in the path using the
-    # provided versions
-    REQUIREMENTS=$(print_conda_requirements)
-    echo "conda requirements string: $REQUIREMENTS"
-    conda create -n testenv --quiet --yes $REQUIREMENTS
-    source activate testenv
+create_new_travisci_env() {
+    REQUIREMENTS=$(echo_requirements_string)
+    pip install $PIP_FLAGS ${REQUIREMENTS}
+    pip install pytest pytest-cov
 
     if [[ "$INSTALL_MKL" == "true" ]]; then
         # Make sure that MKL is used
-        conda install --quiet --yes mkl
-    elif [[ -z $CIRCLECI ]]; then
-        # Travis doesn't use MKL but circle ci does for speeding up examples
-        # generation in the html documentation.
-        # Make sure that MKL is not used
-        conda remove --yes --features mkl || echo "MKL not installed"
+        pip install mkl
     fi
 }
 
+# create_new_conda_env() {
+#     # Skip Travis related code on circle ci.
+#     if [ -z $CIRCLECI ]; then
+#         # Deactivate the travis-provided virtual environment and setup a
+#         # conda-based environment instead
+#         deactivate
+#     fi
+#
+#     # Use the miniconda installer for faster download / install of conda
+#     # itself
+#     wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+#         -O ~/miniconda.sh
+#     chmod +x ~/miniconda.sh && ~/miniconda.sh -b
+#     export PATH=$HOME/miniconda3/bin:$PATH
+#     echo $PATH
+#     conda update --quiet --yes conda
+#
+#     # Configure the conda environment and put it in the path using the
+#     # provided versions
+#     REQUIREMENTS=$(print_conda_requirements)
+#     echo "conda requirements string: $REQUIREMENTS"
+#     conda create -n testenv --quiet --yes $REQUIREMENTS
+#     source activate testenv
+#
+#     if [[ "$INSTALL_MKL" == "true" ]]; then
+#         # Make sure that MKL is used
+#         conda install --quiet --yes mkl
+#     elif [[ -z $CIRCLECI ]]; then
+#         # Travis doesn't use MKL but circle ci does for speeding up examples
+#         # generation in the html documentation.
+#         # Make sure that MKL is not used
+#         conda remove --yes --features mkl || echo "MKL not installed"
+#     fi
+# }
```

Comment:
Why did you add this  ?

Reply:
That was part of the original travis.yml file, I forgot to remove it!